sys/sys/elf_common.h

```diff
@@ -374,7 +374,7 @@ typedef struct {
 #define	EF_PPC_RELOCATABLE_LIB	0x00008000
 
 #define	EF_RISCV_RVC		0x00000001
-#define	EF_RISCV_FLOAT_ABI_MASK	0x00000006
+#define	EF_RISCV_FLOAT_ABI	0x00000006
```

Comment:
This breaks elftoolchain (did you test build this?)

Reply:
only in Android. i was relying on CI for the freebsd testing.